DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/DependencyInjection/RevitConnectorModule.cs

```diff
@@ -49,20 +50,24 @@ public void Load(SpeckleContainerBuilder builder)
     // and where the UoW should be
     // register UI bindings
     builder.AddSingleton<IBinding, TestBinding>();
-    builder.AddSingleton<IBinding, ConfigBinding>("connectorName", "ArcGIS"); // POC: Easier like this for now, should be cleaned up later
+    builder.AddSingleton<IBinding, ConfigBinding>("connectorName", "Revit"); // POC: Easier like this for now, should be cleaned up later
     builder.AddSingleton<IBinding, AccountBinding>();
     builder.AddSingleton<IBinding, BasicConnectorBindingRevit>();
     builder.AddSingleton<IBasicConnectorBinding, BasicConnectorBindingRevit>();
     builder.AddSingleton<IBinding, SelectionBinding>();
     builder.AddSingleton<IBinding, RevitSendBinding>();
-    //no receive?
+    builder.AddSingleton<IBinding, RevitReceiveBinding>();
     builder.AddSingleton<IRevitIdleManager, RevitIdleManager>();
 
     // send operation and dependencies
     builder.AddScoped<SendOperation<ElementId>>();
     builder.AddScoped<IRootObjectBuilder<ElementId>, RevitRootObjectBuilder>();
-
-    // register send conversion cache
     builder.AddSingleton<ISendConversionCache, SendConversionCache>();
+
+    // receive operation and dependencies
+    builder.AddScoped<ReceiveOperation>();
+    builder.AddScoped<IHostObjectBuilder, RevitHostObjectBuilder>();
+    builder.AddScoped<TransactionManager>();
+    builder.AddSingleton(DefaultTraversal.CreateTraversalFunc());
```

Comment:
Yep perfect. This new and improved `DefaultTraversal` is exactly what we should be using (injecting as you have here) from now on.
Tldr, this one should behave functionally equivalent to the old one, while no longer being dependent on the old `CanConvertFunction`.